src/ServerRequest.php

```diff
@@ -117,4 +117,35 @@ public function withoutAttribute($name)
         unset($new->attributes[$name]);
         return $new;
     }
+
+    /**
+     * @internal
+     * @param string $cookie
+     * @return boolean|mixed[]
+     */
+    public static function parseCookie($cookie)
+    {
+        // PSR-7 `getHeaderLine('Cookies')` will return multiple
+        // cookie header comma-seperated. Multiple cookie headers
+        // are not allowed according to https://tools.ietf.org/html/rfc6265#section-5.4
```

Comment:
Should imho read 

... PSR-7 `getHeaderLine('Cookie')` will return multiple comma-seperated cookie headers.